pkg/newlog/cmd/newlogd.go

```diff
@@ -107,6 +108,11 @@ var (
 
 	//domainUUID
 	domainUUID *base.LockedStringMap // App log, from domain-id to appDomain
+	// runtimeAppUUID
+	runtimeAppUUID *base.LockedStringMap // App log, may be nested App logs
```

Comment:
is it maybe easier to just add a boolean field `nested` to `appDomain` struct?

Reply:
that would be a simpler way, yes.